server/views/temporary-accommodation/bedspace-search/results.njk

```diff
@@ -0,0 +1,128 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "../../partials/breadCrumb.njk" import breadCrumb %}
+{% from "../../components/place-context-header/macro.njk" import placeContextHeader %}
+
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = "Search for available bedspaces - " + applicationName %}
```

Comment:
🤔  Would it be worth mentioning how many results are returned in the page title? Ideally the page title should update if the page content could potentially be different ie. search page and results page